src/Formatter.php

```diff
@@ -42,9 +41,9 @@ public static function format(string $content, string $uri)
         $config = new Config(['dummy'], false);
         $config->standards = self::findConfiguration($path);
 
-        // Create this class so it is autoloaded and sets up a bunch
+        // Autoload class to sets up a bunch
         // of PHP_CodeSniffer-specific token type constants.
-        new Tokens();
+        spl_autoload_call('PHP_CodeSniffer\Util\Tokens');
```

Comment:
`Tokens::class` is nicer than string. Btw looked at the class, how can someone think it is a good idea to globally define constants in a class file. Total violation of PSR-2, a file should either define a symbol or cause side-effects. Ironic for a coding-style checker to violate the standard itself